### ISSUE TITLE
Paper: Automatic Recognition of fMRI-Derived Functional Networks Using 3-D Convolutional Neural Networks - IEEE Journals 

### ISSUE THREAD

**Issue by gkiar** [gkiar/reproreading#42]
URL: [https://ieeexplore.ieee.org/document/7949139](https://ieeexplore.ieee.org/document/7949139) 
### This paper does... 

- "Specifically, in this paper, an effective 3D CNN framework with two convolutional layers, one pooling layer and one fully connected layer, was designed for functional network map recognition. "
- resting state maps are of size 45 x 54 x 45

network structure:
- input
- c1: convolution w/ 32 (kernels) x 5 (size) x 2 (stride)
- c2: convolution w/ 32 x 3 x 1
- p: pooling w/ kernel size of 2
- fc: fully connected 128 node layer (for classif)
- output: 10 nodes for RSNs (for classif)
- all convolutions are followed by a relu with 0.1
- training was done with sgd, the loss function as log-likelihood w/ lambda at 0.01
- they used dropout
- cnn layers initialized with Gaussian of mean 0 and sd 0.01
- 80 epochs

overall, this seems like a reasonable place to start for determining network structure of our encoding layer (and inverted for the decoder).
